tomviz/python/AutoTiltAxisRotationAlignment.py

```diff
@@ -87,8 +87,9 @@ def transform_scalars(self, dataset):
         rot_ang = coarseAngles[minIntensityIndex]
 
         self.progress.message = 'Rotating tilt series'
-        result = ndimage.interpolation.rotate(
-            tiltSeries, -rot_ang, axes=((0, 1)))
+        result = np.empty_like(tiltSeries)
```

Comment:
If rot_ang is not a integer multiple of 90 (i.e. 90, 180, 270...), the output's shape will be different from input array.
Same issue for Rotate3D transform.